config/meson.build

```diff
@@ -142,7 +142,7 @@ endif
 # Create the tool chain library as subproject
 mctc_dep = dependency(
   'mctc-lib',
-  version: '>=0.3.0',
+  version: '0.3.2',
```

Comment:
This should have a lower bound of 0.3.0 unless we use an API introduced in a later version.

Reply:
I don't think this is necessary here. Any mctc-lib version after 0.3.0 including head works. But we need at least multicharge v0.3.0 for the actinide tests.